package-lock.json

```diff
@@ -1,943 +1,8 @@
 {
   "name": "kurl",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Is the goal to _not_ explicitly set the package versions so there aren't any conflicts with dependabot proposed changes? What's the difference between ths _packages_ section vs the package.json [_depenedencies_](https://github.com/replicatedhq/kURL/blob/1e21097e1844c1a6f91a46be60b0778075bd00e0/package.json#L9-L13) section?